pycobertura/templates/html-delta.jinja2

```diff
@@ -34,16 +34,16 @@ table.code td.lineno pre {margin-right: 1rem;}
           </tr>
         </thead>
         <tbody>
-{%- for line in lines %}
+{%- for iter in range(lines["Filename"]|length) %}
```

Comment:
Is `iter` the file index in the array? Can we rename it to `file_index` or `file_idx`?

```suggestion
{%- for file_idx in range(lines["Filename"]|length) %}
```

What do you think?

Reply:
done -> renamed to `file_index`